src/wallet/rpcdump.cpp

```diff
@@ -1284,7 +1284,8 @@ static UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, con
                 }
 
                 // add to address book or update label
-                if (IsValidDestination(dest)) {
+                if(!internal) {
+                    assert(IsValidDestination(dest));
                     pwallet->SetAddressBook(dest, label, "receive");
                 }
 
```

Comment:
Missing the changes here 

https://github.com/bitcoin/bitcoin/commit/f617e05c38c9c7e663b824fefe5329cc5cbbf26a#diff-c455c7835ec6d7a6d846c1ce3d3c2ad704b01d0c2215bffb5c28c9a5a1b33369L1105-R1106